test.js

```diff
@@ -23,15 +23,19 @@ function restoreEnvVars(cache) {
 }
 
 test.cb('async', t => {
```

Comment:
Use a normal test and async/await (AVA transpiles tests through Babel)

```js
test('async', async t => {
```

You can simplify the below code to not use `.then`s

Example: https://github.com/sindresorhus/pageres/blob/05457dca42a087cf43f1e0f0d195740dc32a6cba/test/test.js#L71-L76